<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtMetinBotClass</class>
 <widget class="QMainWindow" name="QtMetinBotClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>233</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>529</width>
    <height>233</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>529</width>
    <height>238</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MetinBot</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="metinsKilledLabel">
          <property name="text">
           <string>Metins killed:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="numMetinsKilled">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="tangLabel">
          <property name="text">
           <string>Yang Made:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="numYangMade">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="currYangLabel">
          <property name="text">
           <string>Current Yang:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="currYang">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="deathLabel">
          <property name="text">
           <string>Death Count:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="deathCounter">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>89</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="startButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777214</height>
         </size>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="botModesBox">
        <property name="title">
         <string>Modes</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QRadioButton" name="expButton">
           <property name="text">
            <string>ExpBot</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="farmButton">
           <property name="text">
            <string>FarmBot</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="fishingButton">
           <property name="text">
            <string>FishingBot</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="generalTab">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>10</y>
          <width>231</width>
          <height>131</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Skill Bot</string>
           </property>
           <widget class="QCheckBox" name="activateSkillBotCheckBox">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>23</y>
              <width>59</width>
              <height>17</height>
             </rect>
            </property>
            <property name="text">
             <string>Skill Bot</string>
            </property>
           </widget>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>46</y>
              <width>79</width>
              <height>73</height>
             </rect>
            </property>
            <property name="title">
             <string>Options</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QRadioButton" name="suraSkillsRadioButton">
               <property name="text">
                <string>Sura</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="warriorSkillsRadioButton">
               <property name="text">
                <string>Warrior</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Pickup</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QCheckBox" name="activatePickupCheckBox">
              <property name="text">
               <string>Pickup</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>Options</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <widget class="QCheckBox" name="rangePickupCheckBox">
                 <property name="text">
                  <string>Range Pickup</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>105</width>
          <height>48</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QCheckBox" name="itemUserCheckBox">
           <property name="text">
            <string>Item User</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="runPythonButton">
           <property name="text">
            <string>Execute Python File</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="damageTab">
       <attribute name="title">
        <string>DamageHacks</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCheckBox" name="waitDamageCheckBox">
            <property name="text">
             <string>Wait Damage</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="selectedDamageCheckBox">
            <property name="text">
             <string>Damage Selected</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="securityTab">
       <attribute name="title">
        <string>Security</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>30</y>
          <width>242</width>
          <height>100</height>
         </rect>
        </property>
        <property name="title">
         <string>Silent Options</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QPushButton" name="disableSilentButton">
               <property name="text">
                <string>DisableAll</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="enableSilentButton">
               <property name="text">
                <string>Enable All</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QCheckBox" name="pickupSilentCheckBox">
               <property name="text">
                <string>Range Pickup Silent</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="dmgHackSilentCheckBox">
               <property name="text">
                <string>Damage Selected Silent</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="waitHackSilentCheckBox">
               <property name="text">
                <string>Wait Hack Silent</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="expTab">
       <attribute name="title">
        <string>ExpBot</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="farmTab">
       <attribute name="title">
        <string>FarmBot</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="fishingTab">
       <attribute name="title">
        <string>FishingBot</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtMetinBot.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>activateSkillBotCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_5</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>enableAllSilentOptions()</slot>
 </slots>
</ui>
